packages/db/src/schema.ts

```diff
@@ -69,6 +69,7 @@ export enum Bucket {
   validator_metaData = 41,
 
   backfilled_ranges = 42, // Backfilled From to To, inclusive of both From, To
+  capella_blsToExecutionChange = 56, // ValidatorIndex -> SignedBLSToExecutionChange
```

Comment:
Why 56? We should have some guidelines for picking a bucket number. For now just take the first available, which appears to be 29